tests/end_to_end/complicated_context/irix-g-flags.txt

```diff
@@ -0,0 +1 @@
+--context tests/end_to_end/complicated_context/orig.c
```

Comment:
make this path relative to the test directory and convert it to absolute in the test runner